tap_hubspot/client.py

```diff
@@ -179,12 +179,16 @@ def get_custom_schema(self, poorly_cast: List[str] = []):
 
     def get_properties(self) -> List[dict]:
         response = requests.get(f"{self.url_base}/crm/v3/properties/{self.name}", headers=self.http_headers)
-        res = response.json()
-
         try:
-            return res["results"]
-        except KeyError:
-            raise KeyError(f"Error retrieving the API query results: {res}")
+            res = response.json()
+            response.raise_for_status()
```

Comment:
```suggestion
            res.raise_for_status()
```

This might have been a typo 🤔